components/ContentTemplates/ChartsTemplate.tsx

```diff
@@ -1,5 +1,101 @@
-import { WorkInProgress } from "../WorkInProgress/WorkInProgress"
+import { CodeBlock } from "../CodeBlock/CodeBlock"
+import { NavPage } from "../NavPage/NavPage"
+import { PageUpdated } from "../PageUpdated/PageUpdated"
+import { TemplateSection } from "../TemplateSection/TemplateSection"
+import { chartPageNavigation } from "../../data/pageNavigationLists"
 
 export const ChartsTemplate = () => {
-	return <WorkInProgress />
+	return (
+		<>
+			<NavPage pageNavigation={chartPageNavigation} />
+			<TemplateSection sectionName="introduction" title="Introduction">
+				<p>
+					The chart is a visual representation of data that uses symbols, lines,
+					or bars to convey information. Whether it's comparing features of two
+					mobile devices on a website, results of a survey on a social media
+					post, or a person's monthly progress in their goal, charts should be
+					conveyed in a way that is easy to for all users to read and
+					understand.
+				</p>
+			</TemplateSection>
+			<TemplateSection sectionName="typesOfCharts" title="Types of Charts">
+				<p>
+					Common types of that are used on the web include:
+					<ol>
+						<li>Comparison Charts: Often used to compare two products</li>
+						<li>Bar Charts:Often used to used to compare sale results</li>
+						<li>Line Charts: Often used to convey a company's history</li>
+					</ol>
+				</p>
+			</TemplateSection>
+			<TemplateSection
+				sectionName="howToCreate"
+				title="How to Create a Comparison Chart">
+				<p> Out of all the charts used on websites, comparison charts tend to be
+					used the most. Here's how to add this piece of content to your
+					website.</p>
+				<ol> 
+<li>  Add the table element </li> 
```

Comment:
I would like to keep charts and tables separate so the table information can be moved to the table section.

Reply:
Oh this describes how to build a chart @EmmaDawsonDev